_format/1.0/index.md

```diff
@@ -926,13 +926,13 @@ If an endpoint supports the `include` parameter and a client supplies it,
 the server **MUST NOT** include unrequested [resource objects] in the `included`
 section of the [compound document].
 
+If a server is unable to identify a relationship path or does not support
+inclusion of resources from a path, it **MUST** respond with 400 Bad Request.
+
```

Comment:
I don't think so. A note is not really a cutoff of any sort. Then again, I also fail to see any issue with the current organization.

It occurs to me, though, that the note is somewhat superfluous. The same path is used in the example request that follows soon after. Maybe the substance of the note would be better placed as an ordinary paragraph after the example:

> `GET /articles/1?include=comments.author HTTP/1.1`
> `Accept: application/vnd.api+json`
>
> Here, `comments` is a relationship listed under an `articles` resource object, and `author` is a relationship listed under a `comments` resource object.